eslint-rules/no-single-tests.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(context) {
+    return {
+        Identifier: function(node) {
+            var nodeName = node.name;
+            if (nodeName === 'only') {
+                var parentName = node.parent.object.name;
+                if (parentName === 'it' || parentName === 'describe') {
+                    context.report(node, '.only() was left in a test');
+                }
+            } else if (nodeName === 'fdescribe') {
+                context.report(node, 'fdescribe() was left in a test');
+            } else if (nodeName === 'fit') {
+                context.report(node, 'fit() was left in a test');
```

Comment:
Yep it's really cool how they provide such a nice plugin system to their AST tools. Babel has something similar.

In the jasmine tests f# runs 'only those with f#' so it's like .only but cumulative.